plug-in-development/index.html

```diff
@@ -53,7 +53,7 @@ <h3 class="section" indent="1" id="launch-configuration">Launch configuration</h
 </p>
 <pre class="brush:java">
 fun main(argv: Array&lt;String&gt;) {
-    com.beust.kobalt.main(argv)
+    com.beust.kobalt.internal.main(argv)
```

Comment:
`com.beust.kobalt.main(argv)` should work as long as you have `kobalt.jar` in your classpath (but it's not in `kobalt-plugin-api`).

I confirm it's valid for me if I write it in a project that depends on `kobalt.jar`.